docs/source/developing/releasing.adoc

```diff
@@ -12,15 +12,23 @@ The following sections describe how to release Minishift.
 [[release-prereqs]]
 == Prerequisites
 
-You must have a https://help.github.com/articles/creating-an-access-token-for-command-line-use[GitHub personal access token]
+* You must have a https://help.github.com/articles/creating-an-access-token-for-command-line-use[GitHub personal access token]
 defined in your environment as _GITHUB_ACCESS_TOKEN_.
+* You need _gnutar_ installed on your machine. See also GitHub issue link:https://github.com/minishift/minishift/issues/657[#657].
+
+== Preparing the GitHub milestone
+
+. Verify the milestone you want to release
+ * Move unresolved issues into the next milestone
+ * Review resolved/closed issues and make sure they are classified correctly
+ * Check the resolution of the issues. For example, if some code has changed as part of
```

Comment:
I didn't get this part properly. Example part mostly. What do you mean by "some code has changed as part of the milestone"?

So, this is flow of "resolution" category. 
So, each issue when it's related PR is merged, should be tagged as "resolution/done". If when not done as part of PR merge, then during at this time. (which is above line IMO)
Now, if other issues which is not reproducible, we tag as "resolution/cannot-reproduce".

IMO, line 24 doesn't conveying properly what it should.

Below is my suggestion based on whatever I could understand and can be modified too. 

* Check and update the resolution of the issues accordingly. For example, the issue whose related PR is merged should be labeled as _resolution/done_.